src/main/java/gregtech/api/recipes/RecipeMaps.java

```diff
@@ -197,7 +197,8 @@ public final class RecipeMaps {
     @ZenProperty
     public static final RecipeMap<AssemblyLineRecipeBuilder> ASSEMBLY_LINE_RECIPES = new RecipeMapAssemblyLine<>(
```

Comment:
Could remove the comment about the AL recipe builder not having any special actions.